mongo/tests/test_integration.py

```diff
@@ -174,6 +175,8 @@ def test_integration_replicaset_primary(instance_integration, aggregator, check)
         'collection',
     ]
     _assert_metrics(aggregator, metrics_categories, replica_tags)
+    # Lag metrics are tagged with the state of the member and not with the current one.
+    _assert_metrics(aggregator, ['replset-lag-from-primary'])
```

Comment:
Why do you use hyphens for this metric?

Reply:
This is not a metric, this is a metric_category, i.e metrics are loaded from the json file called `metrics-replset-lag-from-primary.json`